contrib/cf-profile/cf-profile.py

```diff
@@ -0,0 +1,57 @@
+from argparse import ArgumentParser
+import sys
+import json
+import re
+
```

Comment:
Please add a docstring here where you show how the input data looks (an example of a few different events).